### ISSUE TITLE
[SUGGESTION] blocking VP9 only for 1080p/certain resolutions

### ISSUE THREAD

**Issue by EnekoT2001** [alextrv/enhanced-h264ify#26]
1080p VP9 bitrate has been getting lower and lower over the past few years, now youtube offers the quality we used to have on VP9 for premium users as "enhanced bitrate", in the videos that are only 1080p, h264 is preferred (btw 60fps 1080p videos never get the enhanced bitrate treatment).

AV1 looks a bit better than h264 and uses a bit less bitrate to vp9 but it's just so efficient that looks ok enough, problem is that av1 encodings are kinda uncommon still.
So getting an option to block vp9 on 1080p videos would be amazing, this is why i installed the extension to start with tbh

**Comment by travnick** [comment]
True, It would be nice to have an option to use as much HW decode as possible by default (1080p with h264/VP8 instead of VP9 in my case, for nvidia/intel gpu), but do not block higher resolutions with software decoding (have Hi resolution display with enough cpu resources)

**Comment by Araigumayo** [comment]
Wish it does this, now YouTube gives you 1-digit megabytes file (low quality) for a 1-3min 1080p video.
